### PR TITLE
Swift 3

### PR DESCRIPTION
**Pull request by willeeklund**
Thanks for a great repo, I use it in one of my apps.

This PR a Swift 3 update for the framework as well as the example project. Kept them as separate commits, hopefully that makes it easier for you to code review.

Have a great day.

Best regards 
/Wilhelm

### CODE REVIEW

**Comment by Jeraimee** [comment]
👍

File: Pod/Classes/BWWalkthroughViewController.swift [hunk 1]
```diff
@@ -212,64 +205,62 @@ At the moment it's only used to perform custom animations on didScroll.
...
-            
-        }else{
+        } else {
             
             let previousVC = controllers[controllers.count-2]
-            let previousView = previousVC.view;
-            
-            scrollview.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("H:[previousView]-0-[view]", options:[], metrics: nil, views: ["previousView":previousView,"view":vc.view]))
+            if let previousView = previousVC.view {
+                // For this constraint to work, previousView can not be optional
```
Comment (SergeiMeza) [inline Pod/Classes/BWWalkthroughViewController.swift]: I was looking for this bug in the swift3 implementation of this code. Thank you!

**Comment by willeeklund** [comment]
This PR solves #73. 
It would feel great if this could get merged @ariok.

**Comment by ariok** [comment]
Thanks for your work! I'm still OOO for my Honeymoon. I'll be back in a few days and I'll merge these commits.

**Pull request merged**